.github/workflows/test-sdks-remote.yml

```diff
@@ -0,0 +1,70 @@
+name: Test SDKs Remotely
```

Comment:
This workflow runs on pushes to the main branch of sdk-test-data and triggers the testing workflows to run **in** each SDK repository, thus turning them red/green depending on whether they pass any new test cases added to this repo.

Reply:
Do the results factor into the success of _this_ job? Is that what `propagate_failure` does?

Reply:
correct.